src/repositories/players.ts

```
@@ -44,42 +46,32 @@
+    class: ss.string(),
+  });
 
-  if (!("class" in player)) {
-    return [null, "Validation error: player must have a class property"];
-  }
+  const [error, result] = playerDTO.validate(player);
 
-  if (typeof player.class !== "string") {
-    return [null, "Validation error: player class must be a string"];
+  if (error) {
+    return [badRequest("Validation error " + error.message, error), undefined];
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)